res/entities/Player.json

```diff
@@ -4,13 +4,13 @@
     {
       "componentType": "Player"
     },
-	{
+	  { 
       "componentType": "Sprite",
       "src": "entity/player_modelDefault",
-	  "origin": {
-		"x": 18,
-		"y": 48
-	  },
+	    "origin": {
+		    "x": 15,
```

Comment:
What Im for: Merge it and run ALL our source file through some sort of indentation/formatting algorithm, vs has one